### ISSUE TITLE
Allow admins to edit everything about the vendor record

### ISSUE THREAD

**Issue by dwdowns** [johndowns/soscafe#80]
Including Business Name, Email Address and Date Registered (not the ID)

**Comment by JackDownsNZ** [comment]
@johndowns not sure whats needed on the backend to enable this (although I can expect issues with editing email address...)

But front end will be quick to change - so will do once you have had time to check backend.

**Comment by johndowns** [comment]
@dwdowns What do you want to do if the email address changes, with regards to the user role assignments?

**Comment by dwdowns** [comment]
Can we automatically add the new (updated) email address to that table?

www.downwithcancer.nz<http://www.downwithcancer.nz/>
Check out my book here: www.mildtouch.nz<http://www.mildtouch.nz>
Check out my TEDx talk<https://tedxauckland.com/talks/reflections-of-a-genetically-modified-optimist/>.

**Comment by johndowns** [comment]
@dwdowns yes we can - would you want to remove the role assignment for the old one though?

**Comment by dwdowns** [comment]
No, we can leave it there as that's the more common case. I can always go in and remove the record if need be

**Comment by johndowns** [comment]
@JackDownsNZ How will this go on the front-end re the date registered? Any concerns with showing a UI date picker element?

**Comment by JackDownsNZ** [comment]
Nope no concerns there - let me know once the backend is ready and ill make the needed changes on my side

**Comment by JackDownsNZ** [comment]
Working Branch: `admin-updates`

**Comment by dwdowns** [comment]
Can we allow admins to edit the email address on the vendor record too please?

**Comment by johndowns** [comment]
Yes that is already covered by the work we are doing

**Comment by johndowns** [comment]
Admins will be able to edit the business name, email address, and registration date

**Comment by dwdowns** [comment]
Thanks.
I have someone helping me clean crappy data (inc email addresses)

David Downs
david@downsy.co.nz<mailto:david@downsy.co.nz>
021758967
